docs/en/observability/apm/configure/sampling.asciidoc

```diff
@@ -67,9 +67,14 @@ This final policy is used to catch remaining trace events that don't match a str
 === Storage limit
 The amount of storage space allocated for trace events matching tail sampling policies. Caution: Setting this limit higher than the allowed space may cause APM Server to become unhealthy.
 
+A value of `0` or equivalent (e.g. `0GB`) means unlimited local tail sampling database size
```

Comment:
[to reviewer] Feel free to improve the description. I'm sure there's a better way to express this.

Reply:
The _unlimited ... but 80%_ confuses me a bit; how about tweaking a bit like:

_A value of `0GB` does not set a concrete limit, but rather allows the APM Server to align its disk usage with the disk size. The server uses up to a 80% of the disk usage limit on the disk where the database is located. The last 20% of disk will not be used by APM Server. It is the recommended value as it automatically scales with the disk size._

_If this is not desired, a concrete `GB` value can be set for the maximum amount of disk used for Tail Based Sampling._